.flowconfig

```diff
@@ -3,7 +3,7 @@
 .*/node_modules/electron-packager/test/fixtures/infer-malformed-json/package.json
 
 # Graphql contains a typing error --> ignore it
-.*/node_modules/graphql/**/*.js
+.*/node_modules/graphql/**/**/*.js
```

Comment:
Not sure why this was required. But with-out it flow failed with many errors for graphql.
I think the `**` glob should match any subdirectories as well but it wasn't working.